src/pg/sql/45_observatory_mvt.sql

```diff
@@ -263,17 +263,79 @@ BEGIN
 END
 $$ LANGUAGE plpgsql PARALLEL RESTRICTED;
 
+CREATE OR REPLACE FUNCTION cdb_observatory.OBS_DecodeMCCategory(category TEXT)
```

Comment:
Maybe is a bit hardcore to create a function to return this. What do you think?

Reply:
Totally. I'm removing it :+1: